string requester
string model
int32 size
string color
int32 x
int32 y
int32 customer_id
---
bool accepted